h2o-algos/src/main/java/hex/glrm/GLRM.java

```diff
@@ -622,6 +622,7 @@ public void computeImpl() {
         model._output._iterations = 0;
         model._output._updates = 0;
         model._output._avg_change_obj = 2 * TOLERANCE;    // Allow at least 1 iteration
+        model._output._step_size = 0;       // set to zero
```

Comment:
I would say you need to announce this change in #api-updates since this is a new output parameter